overlay/make-package-set/simplified.nix

```
@@ -7,7 +7,7 @@
 args@{
   rustChannel,
   packageFun,
-  packageOverrides ? _: [ ],
+  packageOverrides ? _: rustBuilder.overrides.all,
```

Let's use the overrides from the `pkgs` argument here. This `packageOverrides` is different from the one in `full.nix`, as it's a function and is called twice, once with `pkgs` and once with `buildPackages`.

```suggestion
  packageOverrides ? pkgs: pkgs.rustBuilder.overrides.all,
```

---

Love it! Let's do that.